server/jobs/job_store.go

```diff
@@ -19,6 +21,11 @@ type Job struct {
 	Status JobStatus
 }
 
+func (j *Job) GetReader() (io.Reader, int64) {
+	logs := strings.Join(j.Output, "\n")
+	return strings.NewReader(logs), int64(len(logs))
```

Comment:
I don't think `len(logs)` is the right size here. It is just length of the array. Looks like stow is expecting the size of the file.

Reply:
logs is the string built after joining the logs slice with newlines so `len(logs)`should give the size of the file to be uploaded. The naming is confusing now that I think of it. I've renamed the variable to `logString`.

Reply:
oh, you're right. I was reading the other way, I swear I don't have dyslexia lol